skynet/types.py

```diff
@@ -0,0 +1,230 @@
+from enum import StrEnum
+
+from msgspec import Struct
+
+
+class ModelMode(StrEnum):
+    DIFFUSE = 'diffuse'
+    TXT2IMG = 'txt2img'
```

Comment:
for some clarity for me, why do we have a separate `DIFFUSE`?

Reply:
Like i already explained, compatibility with current requests sent by frontends for example here is current last request sent by discord: https://explorer-test.telos.net/transaction/57051c8e1e6c7dd5f062d27e2160307827a7ff46f82452d02aec2850fae956bf